docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,542 @@
+
+Runtime filter is a common optimization to improve join performance. It is designed to dynamically generate filter 
+conditions for certain Join queries at runtime to reduce the amount of scanned or shuffled data, avoid unnecessary I/O 
+and network transmission, and speed up the query. We introduced runtime filters in Flink 1.18, and verified its 
+effectiveness through the TPC-DS benchmark, and observed up to 3x speedup for some queries by enabling this feature.
+
+#### Operator Fusion Codegen for Flink SQL
+
+Operator Fusion Codegen improves the execution performance of a query by fusing an operator DAG into a single optimized 
+operator that eliminates virtual function calls, leverages CPU registers for intermediate data and reduces the 
+instruction cache miss. As a general technical optimization, we verified its effectiveness through TPC-DS, and 
+only some batch operators completed fusion codegen support in version 1.18, getting significant performance gains on 
+some query.
+
+Note that both features are considered experimental and disabled by default in Flink 1.18. 
+They can be enabled using `table.optimizer.runtime-filter.enabled` and `able.exec.operator-fusion-codegen.enabled` 
+respectively.
+
+Since Flink 1.16, the Apache Flink Community has been continuously tracking the performance of its batch engine via the 
+TPC-DS benchmarking framework. After significant improvements in Flink 1.17 (dynamic join-reordering, 
+dynamic local aggregations), the two improvements described in the previous sections lead to 12% performance improvement
+compared to Flink 1.17 , a 35% performance improvement compared to Flink 1.16 on a 10T dataset for partitioned tables.
+
+<div style="text-align: center;">
+<img src="/img/blog/2023-10-10-release-1.18.0/tpc-ds-benchmark.png" style="width:90%;margin:15px">
+</div>
+
+**More Information**
+* [FLIP-324: Introduce Runtime Filter for Flink Batch Jobs](https://cwiki.apache.org/confluence/display/FLINK/FLIP-324%3A+Introduce+Runtime+Filter+for+Flink+Batch+Jobs)
+* [FLIP-315: Support Operator Fusion Codegen for Flink SQL](https://cwiki.apache.org/confluence/display/FLINK/FLIP-315+Support+Operator+Fusion+Codegen+for+Flink+SQL)
+
+# Towards Cloud-Native Elasticity
+
+Elasticity describes the ability of a system to adapt to workload changes in a non-disruptive ideally automatic manner.
+It is a defining characteristic of cloud-native systems and for long-running streaming workloads it is particularly 
+important. As such elasticity improvements are an area of continuous investment in the Apache Flink community. 
+Recent initiatives include the Kubernetes 
+[Autoscaler](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-1.6/docs/custom-resource/autoscaler/), 
+numerous improvements to rescaling performance and last but not least 
+the [Adaptive Scheduler](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/deployment/elastic_scaling/#adaptive-scheduler).
```

Comment:
I assume we will change the docs link to be 1.18 in the final release notes

Reply:
Yes, I will do a find+replace.